tests/ha/ha_cluster_join.pm

```diff
@@ -30,7 +30,7 @@ sub run {
     assert_screen 'ha-cluster-join-password';
     type_password;
     send_key 'ret';
-    wait_still_screen;
+    wait_still_screen(stilltime => 10);
```

Comment:
Why is the default 7s not enough?

Reply:
I had sporadic failures with the default value of 7s (on my x86_64 lab), but none after this change.